packages/calcite-components/src/components/list-item/list-item.scss

```diff
@@ -1,6 +1,21 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-list-background-color-hover: Specifies the component's background color when hovered.
+ * @prop --calcite-list-background-color-press: Specifies the component's background color when pressed.
```

Comment:
Similar to above, does this need to be “list-item” - what hover would we actually be theming here on the parent?

Reply:
we are limiting tokens when shared between grouped components. This is documented in the wiki.

Reply:
Yeah it is a bit weird to be targeting the parent hovering when its really the items being hovered.